pages/login.js

```diff
@@ -0,0 +1,30 @@
+import Image from 'next/image';
+
+import Background from '@/components/Background';
+import LoginForm from '@/components/forms/Login';
+
+import logo from '../public/logo.png';
+
+export const loginTestIds = {
+  logInText: 'log-in-text',
+};
```

Comment:
How about renaming this a bit to make it more descriptive? maybe something like this 💡 

```suggestion
export const loginPageTestIds = {
  headlineText: 'headline-text',
};
```

Reply:
done in https://github.com/tyrro/react-survey/pull/50/commits/ac569c09a1643adb280fe4d6a3705d1c347ab318